overrides/engine/prefabs/defaultVisualCharacter.prefab

```diff
@@ -0,0 +1,16 @@
+{
+    "SkeletalMesh" : {
```

Comment:
In most of the existing codebase, the format used is
`"attributeName": {`

It's fine even if you don't want to follow that (not recommended, but some people do have their own styles), but in this file itself you use
`"attributeName" : {`
`"attributeName" :{`
`"attributeName": {`

Consistency please :)